instrumentation-genai/opentelemetry-instrumentation-openai-v2/README.rst

```diff
@@ -17,6 +17,53 @@ Installation
 
     pip install opentelemetry-instrumentation-openai-v2
 
+Usage
+-----
+
+Instrumenting all clients
+*************************
+
+When using the instrumentor, all clients will automatically trace OpenAI chat completion operations
+and capture prompts and completions as events.
```

Comment:
This only happens when `OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT` is set as `true`, right? Can we highlight the parameter here as well?

Reply:
Might be worthwhile putting the example (json) format of the content.

Also might be worth stating that if doing this you may have to configure your pipeline to redact any sensitive info contained in the content.

Reply:
All examples can be found in semantic conventions - https://github.com/open-telemetry/semantic-conventions/blob/main/docs/gen-ai/gen-ai-events.md and it'd be hard to keep them in sync. 

I don't think we should have a redaction example specific to OpenAI instrumentation.